bin/logstash.lib.sh

```diff
@@ -90,7 +91,17 @@ setup_java() {
   if [ -z "$JAVACMD" ]; then
     setup_bundled_jdk_part
     JAVACMD_TEST=`command -v java`
-    if [ -n "$JAVA_HOME" ]; then
+    if [ -n "$LS_JAVA_HOME" ]; then
+      echo "Using LS_JAVA_HOME defined java: ${LS_JAVA_HOME}"
+      if [ -x "$LS_JAVA_HOME/bin/java" ]; then
+        JAVACMD="$LS_JAVA_HOME/bin/java"
+        if [ -d "${LOGSTASH_HOME}/${BUNDLED_JDK_PART}" -a -x "${LOGSTASH_HOME}/${BUNDLED_JDK_PART}/bin/java" ]; then
+          echo "WARNING, using LS_JAVA_HOME while Logstash distribution comes with a bundled JDK"
```

Comment:
I believe the format for warnings is "WARNING: <message>". You are using a ',' instead. So should be:

WARNING: Using LS_JAVA_HOME env variable while Logstash distribution comes with a bundled JDK (<version>)"

I made a couple of minor adjustments. Would be nice to specify which version of the JDK is bundled (the full version). Lets use end of sentence punctuation here as well.

Reply:
One way to print the full version of the bundled JDK could be to execute the `"${LOGSTASH_HOME}/${BUNDLED_JDK_PART}/bin/java -version" `. But this produces 3 lines of output after the "WARNING:" line, it's not inlined.